util/sliceutil/contains.go

```diff
@@ -0,0 +1,11 @@
+package sliceutil
+
+func Contains[T comparable](array []T, value T) bool {
+	for _, item := range array {
+		if item == value {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
May you please use this approach instead: 
```Golang
import  "slices"

things := []string{"foo", "bar", "baz"}
slices.Contains(things, "foo") // true
```

in kobler.go: `if !slices.Contains(request.Cur, supportedCurrency) {` 
Tests pass with this change. 

Existing usages for the reference: `account_validator.go`: `if accountInfo != nil && slices.Contains(cfg.AccountFilter.AllowList, accountInfo.Id) {`

Reply:
Thank you for this tip, removed the whole Contains util in favour of this solution